### ISSUE TITLE
Can not reproduce results on VOC dataset after base training.

### ISSUE THREAD

**Issue by Jaraxxus-Me** [YoungXIAO13/FewShotDetection#41]
Hi, thanks for your work.
I tried to reproduce the results on VOC2007 test dataset, but got confused.
First, I tried to finetune the model provided, which is base trained. I got similar results as reported.
Second, I tried to train from base stage to fine tune stage and I got much worse results, mAP nearly 0. I tried 3 times parallel training, none of the attempt worked well. (first split) All steps were following the README file.
Can you help explain this problem? Thanks.

**Comment by hanhao2019102935** [comment]
@Jaraxxus-Me Can you tell me set?how you built this VOC data ？I am also very surprised that this question arises.

**Comment by Jaraxxus-Me** [comment]
Hi, thanks so much for your response.
About VOC dataset, I downloaded from this mirror [web](https://pjreddie.com/projects/pascal-voc-dataset-mirror/) since the official one seems to be down. Then just follow your instruction in the README file.
I've carefully inspected this problem today. I tried to continue train both your provided model and my self-trained model in the first stage (see attached figures). The loss of your model seems to be much lower (0.3) than mine (1.5), and my meta loss seems to remain 0.1806.
Then I re-implemented my experiment again from the beginning. Found that the loss seems to remain around 1.5 since the forth epoch. Even more wired is that, the meta loss in all 3 splits seem to remain 0.1806 after several epochs (higher than yours).
I'm still stuck on this problem.
Can you offer me some advice maybe?
Thanks.
![training_provided_model](https://user-images.githubusercontent.com/56875617/121684178-ee113900-caf0-11eb-8769-aef0c415b965.png)
![training_self-trained_model](https://user-images.githubusercontent.com/56875617/121684193-f2d5ed00-caf0-11eb-94a9-5a69aaef8f8e.png)

**Comment by Jaraxxus-Me** [comment]
Besides, I randomly got this situation, which continues to confuse me. I have to try several times before getting
![Screenshot from 2021-06-11 20-18-18](https://user-images.githubusercontent.com/56875617/121685386-73491d80-caf2-11eb-8497-b93afb38dbfc.png)
 it normal. I didn't modify your code. After building up the dataset, I simply ran the .sh files.

**Comment by qindengda** [comment]
> Hi, thanks so much for your response.
> About VOC dataset, I downloaded from this mirror [web](https://pjreddie.com/projects/pascal-voc-dataset-mirror/) since the official one seems to be down. Then just follow your instruction in the README file.
> I've carefully inspected this problem today. I tried to continue train both your provided model and my self-trained model in the first stage (see attached figures). The loss of your model seems to be much lower (0.3) than mine (1.5), and my meta loss seems to remain 0.1806.
> Then I re-implemented my experiment again from the beginning. Found that the loss seems to remain around 1.5 since the forth epoch. Even more wired is that, the meta loss in all 3 splits seem to remain 0.1806 after several epochs (higher than yours).
> I'm still stuck on this problem.
> Can you offer me some advice maybe?
> Thanks.
> ![training_provided_model](https://user-images.githubusercontent.com/56875617/121684178-ee113900-caf0-11eb-8769-aef0c415b965.png)
> ![training_self-trained_model](https://user-images.githubusercontent.com/56875617/121684193-f2d5ed00-caf0-11eb-94a9-5a69aaef8f8e.png)

Hi，have you solved the problem? I have the same problems as you.

**Comment by Jaraxxus-Me** [comment]
> > Hi, thanks so much for your response.
> > About VOC dataset, I downloaded from this mirror [web](https://pjreddie.com/projects/pascal-voc-dataset-mirror/) since the official one seems to be down. Then just follow your instruction in the README file.
> > I've carefully inspected this problem today. I tried to continue train both your provided model and my self-trained model in the first stage (see attached figures). The loss of your model seems to be much lower (0.3) than mine (1.5), and my meta loss seems to remain 0.1806.
> > Then I re-implemented my experiment again from the beginning. Found that the loss seems to remain around 1.5 since the forth epoch. Even more wired is that, the meta loss in all 3 splits seem to remain 0.1806 after several epochs (higher than yours).
> > I'm still stuck on this problem.
> > Can you offer me some advice maybe?
> > Thanks.
> > ![training_provided_model](https://user-images.githubusercontent.com/56875617/121684178-ee113900-caf0-11eb-8769-aef0c415b965.png)
> > ![training_self-trained_model](https://user-images.githubusercontent.com/56875617/121684193-f2d5ed00-caf0-11eb-94a9-5a69aaef8f8e.png)
> 
> Hi，have you solved the problem? I have the same problems as you.

Unfortunately not, I still don't know why and have to find another code base for my project..
